geany.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <pkgname>geany</pkgname>
+  <name>Geany Integrated Development Environment</name>
+  <project_license>GPL-2.0+</project_license>
+  <summary>A fast and lightweight IDE using GTK+</summary>
+  <url type="homepage">http://geany.org/</url>
+  <description>
+    <p>Geany is a small and lightweight integrated development environment.
+It was developed to provide a small and fast IDE, which has only a
+few dependencies from other packages. Another goal was to be as
+independent as possible from a special Desktop Environment like KDE
+or GNOME. So it is using only the GTK2 toolkit and therefore you need
+only the GTK2 runtime libraries to run Geany.</p>
+  </description>
+  <screenshots>
+<!-- Screenshots need to be 16:9 ratio but the ones here from the main website are not. Please
```

Comment:
your document states "Ideally, all screenshots should have a 16:9 aspect ratio", nothing formally required.